### DISCUSSION TITLE
Akvorado is lagging

### DISCUSSION THREAD

**Discussion by Faeez-Ali** [akvorado/akvorado#561] [Q&A]
I am running Standalone Akvarado. it sometimes lag. what could be the reason and how could i resolve this?

thanks is advance

![image](https://user-images.githubusercontent.com/64612272/224697719-6a5c7357-d5b0-4b86-bc0a-f31bb0e03264.png)

**Comment by doup123** [comment]
I am facing the same issue. There are various times in the day in which I get the same image as above.
Additionally, for that period at which we face that behaviour there are no data at the gui. However when akvorado is operational again those data are fine. In that case, I am facing a huge lag based on the provided way to calculate it. @vincentbernat any ideas?

**Reply by vincentbernat** [reply]
It looks like that's ClickHouse ingestion which is lagging behind. You need more CPU and/or memory for ClickHouse to avoid that.
